source/declarations.tex

```diff
@@ -879,33 +879,26 @@
 
 \begin{example}
 \begin{codeblock}
-constexpr int square(int x)
-  { return x * x; }             // OK
-constexpr long long_max()
-  { return 2147483647; }        // OK
-constexpr int abs(int x) {
-  if (x < 0)
-    x = -x;
-  return x;                     // OK
+constexpr int square(int x) {           // OK
+  return x * x;
 }
-constexpr int constant_non_42(int n) {  // OK
-  if (n == 42) {
-    static int value = n;
-    return value;
-  }
-  return n;
+constexpr std::generator<int> iota(int n = 0) {
+    while (true) co_yield n++;          // error: \tcode{co_yield} cannot be used in a constexpr function
```

Comment:
I think this example can be improved.
[expr.const] bans `co_await` and `co_yield`,
but [dcl.constexpr]p3 (the paragraph the example is attached to)
bans a constexpr function that is a coroutine.
So I think it might better to use `co_return` instead,
which isn't banned by [expr.const].